exercises/bowling/bowling_test.exs

```diff
@@ -0,0 +1,152 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("bowling.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule BowlingTest do
+  use ExUnit.Case
+
+  def roll_from_array(game, []) do game end
```

Comment:
Usually we put helper functions at the end of the test file (in Ruby), I don't know if Elixir has the same practice.